src/Storage/Validator/FileExt.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Utopia\Storage\Validator;
+
+use Utopia\Validator;
+
+class FileExt extends Validator
+{
+    const TYPE_JPEG = "jpeg";
+    const TYPE_JPG = "jpg";
+    const TYPE_GIF = 'gif';
```

Comment:
You have to decide on a type of quotation marks 😉

btw, should there be more file extensions added to this - or is this just default examples?

Reply:
Fixed quotation marks. I figured the list is endless, so we'll just add popular ones when we need them.